test_util/marathon.py

```diff
@@ -9,7 +9,17 @@
 
 import test_util.helpers
 
+DEFAULT_API_BASE = '/marathon/v2'
```

Comment:
I'd rather have the api base be `/marathon`, people are familiar with `v2/apps` and friends as the API endpoints they call, rather than just "apps".